neo/io/nwbio.py

```diff
@@ -458,15 +476,16 @@ def write_all_blocks(self, blocks, **kwargs):
                 'block', 'the name of the Neo Block to which the SpikeTrain belongs')
 
         if sum(statistics(block)["Epoch"]["count"] for block in blocks) > 0:
-            nwbfile.add_epoch_column('_name', 'the name attribute of the Epoch')
-            # nwbfile.add_epoch_column('_description', 'the description attribute of the Epoch')
-            nwbfile.add_epoch_column(
-                'segment', 'the name of the Neo Segment to which the Epoch belongs')
-            nwbfile.add_epoch_column('block',
-                                     'the name of the Neo Block to which the Epoch belongs')
+            nwbfile.add_trial_column('segment', 'name of the Segment to which the Epoch belongs')
+            nwbfile.add_trial_column('block', 'name of the Block to which the Epoch belongs')
 
+        arr = [[], []]  # epoch array for ascending t_start and t_stop
         for i, block in enumerate(blocks):
-            self.write_block(nwbfile, block)
+            block_name = block.name
+            self.write_block(nwbfile, block, arr)
+        arr2 = np.sort(arr)
+        self._write_epoch(nwbfile, arr2, block)
```

Comment:
`self.write_block` and `self._write_epoch` receive the same arguments, but in different order. Maybe this would be good to harmonize?